package.json

```diff
@@ -78,6 +78,7 @@
     "gulp-sass": "3.1.0",
     "gulp-sourcemaps": "2.4.1",
     "gulp-stylelint": "3.9.0",
+    "gulp-theo": "^1.0.0-alpha.4",
```

Comment:
@NicolaasDieBaas Not sure. @imsickofmaps was this part of what you added to run the styles to write to index.css?